squad-server/layers/layer.js

```diff
@@ -20,23 +20,11 @@ export default class Layer {
     for (const t of ['team1', 'team2']) {
       let team = data[t] || data.teamConfigs[t];
       this.teams.push({
-        faction: team.faction,
-        name: team.teamSetupName,
+        faction: team.defaultFactionUnit, 
+        name: team.defaultFactionUnit, // There is no other name presented in the data so in order to keep the same structure to support the old formatted codes, used the same name
         tickets: team.tickets,
-        commander: team.commander,
-        vehicles: (team.vehicles || []).map((vehicle) => ({
-          name: vehicle.type,
```

Comment:
Isn't there some custom plugins that use this data still?

Reply:
I think dave's mapvote uses to show what the layer have as vehicles in the broadcast. But since the new layer list doesn't provide this info anymore we can't get that

Reply:
best thing for custom plugins would be set it as empty array